contract_tests/test_integration_queue.py

```diff
@@ -728,6 +794,20 @@ def test_if_get_existing_request_then_return_200_and_response_with_correct_param
         assert response.json()["errorCode"] is None
         assert response.json()["errorMessage"] is None
 
+    def test_if_get_existing_request_with_validation_errors_then_return_200_and_response_with_correct_params(self, grundsteuer_data_with_validation_errors):
+        response = requests.post(ERICA_TESTING_URL + self.endpoint,
+                                 data=json.dumps(grundsteuer_data_with_validation_errors, default=json_default))
+        assert response.status_code == 201
+        uuid = response.headers['Location'].split("/")[2]
+        sleep(6)
+        response = requests.get(ERICA_TESTING_URL + self.endpoint + "/" + uuid)
+        assert response.status_code == 200
+        assert response.json()["processStatus"] == "Failure"
+        assert response.json()["result"] is not None
+        assert response.json()["result"]["validation_errors"] is not None
+        assert response.json()["errorCode"] is not None
+        assert response.json()["errorMessage"] is not None
+
```

Comment:
Could we maybe have a similar test for est? I can also add that if you want me to

Reply:
I already added it one commit prior:https://github.com/digitalservice4germany/erica/pull/110/commits/7cae9d0cda6f042d82ef1850017c56dd386b1a70